roles/runtime/tasks/initialize_base.yml

```diff
@@ -188,7 +188,42 @@
 - name: Prepare for CDP DW experiences
   when: run__include_dw
   block:
-    - name: Retrieve CDP DW experiences
-      cloudera.cloud.dw_cluster_info:
-        env: "{{ run__env_name }}"
-      register: run__dw_list
\ No newline at end of file
+    - name: Construct CDP DW Data catalog configurations
+      ansible.builtin.set_fact:
+        run__dw_dbc_configs: "{{ run__dw_dbc_configs | default([]) | union([config]) }}"
+      vars:
+        include: "{{ lookup('template', __dw_config.include | default('experiences_config_placeholder.j2')) | from_yaml }}"
+        use_default_dbc: "{{ __dw_config.use_default_dbc | default(False) | bool }}"
```

Comment:
Would we not want to use the default catalog if the user doesn't specify one?
Perhaps it is preferred to create a new explicitly named catalog rather than share the default?

Reply:
I don't have a strong preference either way. I felt it was better with the current approach, in case the user is not too bothered with giving a unique to the DBCs and VWs and I saw that in the playbooks for other experiences we let ansible generate a unique name for their services, maybe I thought that was the preferred way.

Reply:
yeah we try to generate a unique name by default from the name_prefix where the object isn't a 1:1 mapping to the environment, or otherwise needs some source of uniqueness, like with datahub names.
For CDW, I believe the K8s cluster is 1:1 with the Environment, and will always have a default DBC, so I think it would be better if we used the default DBC as the Ansible default, and only worried about unique names for the VWs.
If that isn't workable for some reason I'm not particularly wedded to the idea, my thought is that the default DBC is less hassle to clean up.